packages/core/src/sass/app/modules/cashier.scss

```diff
@@ -233,7 +233,7 @@
             }
             @include mobile {
                 min-height: auto;
-                min-width: auto;
+                min-width: calc(100% - 4rem);
```

Comment:
pls help to add comment on what is 4rem here 🙏

Reply:
@mustofa-binary I found this `calc(100% - 4rem)` as basic of dropdown select picker in deriv.app CSS. do I need to add a comment on it?

Reply:
got it. its fine as it is 👍